src/python/pants/bin/pants_loader.py

```diff
@@ -9,6 +9,7 @@
 import warnings
 from textwrap import dedent
 
+from pants import ox  # noqa: F401
```

Comment:
Is this to force the `ox` module to be in the binary? It might also be reasonable to use an explicit dep in the `BUILD` file. Either way, a comment would be good.

EDIT: ...aha, this is for import sideeffects. Yea, definitely definitely a comment, or possibly an explicit method instead.

Reply:
I think a method is the best way to avoid having to write all those warning ignorators. Doing that instead.